app/views/plants/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Update Plant Information</h1>
 <div class="form_container">
-<%= form_for @plant do |f| %>
+<%= form_for(@plant, multiplart: true) do |f| %>
```

Comment:
should this be `multiplart`? Not `multipart`?

Reply:
![paul-blart](https://user-images.githubusercontent.com/42391567/53134914-64f82180-3536-11e9-90bf-070bf5e5e2bd.png)